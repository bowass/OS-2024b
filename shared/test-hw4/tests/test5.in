64328
16289
6443
14854
1620
53123
291
42075
28580
52741
62311
61329
55523
23788
61275
25533
32512
57142
61819
49829
63958
7073
26814
18412
11655
34456
4303
58277
37161
45143
62449
30469
16153
18122
64102
8170
42485
53273
59704
33334
49081
65032
55695
6198
16863
21230
18295
31465
31390
446
23547
8986
32743
54845
38176
65507
47173
57787
15253
13149
48149
7058
54786
31392
21804
50077
9396
27194
5521
11225
41244
65086
42927
22675
35759
61845
47233
57529
18576
15747
63136
18190
58870
1199
46394
45900
19488
58916
39608
45877
39736
42245
31587
31774
39477
42003
30990
13165
2097
16108
39133
23378
8601
64190
19889
9079
62168
55264
28126
8443
16804
17393
10065
38667
23491
27283
47488
38126
62310
29472
27909
426
8667
9419
37953
13671
53378
22850
33347
37980
43202
63830
32623
34823
16451
29028
56987
26974
7037
48481
37893
26499
25042
57428
39335
3949
23936
9067
1489
64865
7304
12092
44589
34507
58377
34508
35890
8597
45455
41341
25192
43599
50196
43755
54257
50039
42355
11932
48930
20987
49522
36290
61858
48566
33583
64067
17709
46062
35677
25731
22711
2240
39570
14041
6065
8606
13750
23976
11624
5816
57488
14954
27734
11913
65155
57326
20025
39374
23107
23986
51334
15502
46642
64120
30396
58308
25094
4870
45411
26726
21485
34629
28196
1793
35095
55637
52347
5781
32969
41800
56704
44254
16366
65067
21750
23204
18912
15918
52866
48542
22760
4037
11551
57915
58716
27550
41293
26083
29422
53530
30369
1483
12881
16006
40529
62702
48037
811
58686
32892
12364
33511
14775
31559
42707
59157
33822
58182
59108
43638
34458
24634
750
41056
64365
36948
31123
40942
42406
47136
40919
48948
13848
21168
46771
35929
28601
40919
5636
25092
27010
51970
48231
59032
21713
2516
22331
60593
26305
55
11602
48486
24084
25797
45214
40843
62187
53550
7096
9187
65348
26539
35123
62644
39192
19501
18364
5316
65070
59101
52246
37579
53593
63705
54469
51087
26533
9621
63383
30841
33594
16040
5917
21105
25668
62026
32744
35976
29197
50897
36940
56216
12705
50078
33022
60855
27793
15089
48155
51942
15287
51853
45309
22290
36212
3892
5559
61883
31971
53744
59333
62132
23466
54076
54211
49400
57750
19228
55031
11480
26439
61997
4217
23963
61987
10908
22557
40688
19264
45387
55275
14154
13004
23018
54244
43524
2447
48750
65505
33617
2245
18631
63130
22809
53704
23564
42225
20456
853
33393
16432
34303
39187
60915
10259
3988
57717
60324
21946
58495
10184
54538
53807
3657
43799
11693
36648
3458
49363
63808
23335
34196
61253
20814
13145
56779
26966
32294
8981
9973
34187
39141
45427
2383
26919
21340
1582
61375
29458
45992
44730
28542
11262
39571
47387
57781
31859
12719
64018
60358
14307
49075
27585
33142
21727
24199
52045
54270
50900
40311
48658
14051
37756
1496
13012
26447
4438
27645
63763
46549
45538
24612
26601
60983
19961
27175
22498
40372
6182
63751
60182
53917
55752
48730
10072
50844
28006
11551
36424
36813
58036
62340
57562
64750
29908
41388
54333
61382
39496
14659
23337
47765
42681
40337
38501
50610
39104
48015
16336
10675