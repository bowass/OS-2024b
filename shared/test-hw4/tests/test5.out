Virtual address: 64328 Physical address: 72 Value: 0
Virtual address: 16289 Physical address: 417 Value: 0
Virtual address: 6443 Physical address: 555 Value: 74
Virtual address: 14854 Physical address: 774 Value: 14
Virtual address: 1620 Physical address: 1108 Value: 0
Virtual address: 53123 Physical address: 1411 Value: -32
Virtual address: 291 Physical address: 1571 Value: 72
Virtual address: 42075 Physical address: 1883 Value: 22
Virtual address: 28580 Physical address: 2212 Value: 0
Virtual address: 52741 Physical address: 2309 Value: 0
Virtual address: 62311 Physical address: 2663 Value: -39
Virtual address: 61329 Physical address: 2961 Value: 0
Virtual address: 55523 Physical address: 3299 Value: 56
Virtual address: 23788 Physical address: 3564 Value: 0
Virtual address: 61275 Physical address: 2907 Value: -42
Virtual address: 25533 Physical address: 3773 Value: 0
Virtual address: 32512 Physical address: 3840 Value: 0
Virtual address: 57142 Physical address: 4150 Value: 55
Virtual address: 61819 Physical address: 4475 Value: 94
Virtual address: 49829 Physical address: 4773 Value: 0
Virtual address: 63958 Physical address: 5078 Value: 62
Virtual address: 7073 Physical address: 5281 Value: 0
Virtual address: 26814 Physical address: 5566 Value: 26
Virtual address: 18412 Physical address: 5868 Value: 0
Virtual address: 11655 Physical address: 6023 Value: 97
Virtual address: 34456 Physical address: 6296 Value: 0
Virtual address: 4303 Physical address: 6607 Value: 51
Virtual address: 58277 Physical address: 6821 Value: 0
Virtual address: 37161 Physical address: 6953 Value: 0
Virtual address: 45143 Physical address: 7255 Value: 21
Virtual address: 62449 Physical address: 2801 Value: 0
Virtual address: 30469 Physical address: 7429 Value: 0
Virtual address: 16153 Physical address: 281 Value: 0
Virtual address: 18122 Physical address: 7882 Value: 17
Virtual address: 64102 Physical address: 8038 Value: 62
Virtual address: 8170 Physical address: 8426 Value: 7
Virtual address: 42485 Physical address: 8693 Value: 0
Virtual address: 53273 Physical address: 8729 Value: 0
Virtual address: 59704 Physical address: 9016 Value: 0
Virtual address: 33334 Physical address: 9270 Value: 32
Virtual address: 49081 Physical address: 9657 Value: 0
Virtual address: 65032 Physical address: 9736 Value: 0
Virtual address: 55695 Physical address: 10127 Value: 99
Virtual address: 6198 Physical address: 10294 Value: 6
Virtual address: 16863 Physical address: 10719 Value: 119
Virtual address: 21230 Physical address: 10990 Value: 20
Virtual address: 18295 Physical address: 5751 Value: -35
Virtual address: 31465 Physical address: 11241 Value: 0
Virtual address: 31390 Physical address: 11166 Value: 30
Virtual address: 446 Physical address: 1726 Value: 0
Virtual address: 23547 Physical address: 11515 Value: -2
Virtual address: 8986 Physical address: 11546 Value: 8
Virtual address: 32743 Physical address: 4071 Value: -7
Virtual address: 54845 Physical address: 11837 Value: 0
Virtual address: 38176 Physical address: 12064 Value: 0
Virtual address: 65507 Physical address: 12515 Value: -8
Virtual address: 47173 Physical address: 12613 Value: 0
Virtual address: 57787 Physical address: 12987 Value: 110
Virtual address: 15253 Physical address: 13205 Value: 0
Virtual address: 13149 Physical address: 13405 Value: 0
Virtual address: 48149 Physical address: 13589 Value: 0
Virtual address: 7058 Physical address: 5266 Value: 6
Virtual address: 54786 Physical address: 11778 Value: 53
Virtual address: 31392 Physical address: 11168 Value: 0
Virtual address: 21804 Physical address: 13868 Value: 0
Virtual address: 50077 Physical address: 14237 Value: 0
Virtual address: 9396 Physical address: 14516 Value: 0
Virtual address: 27194 Physical address: 14650 Value: 26
Virtual address: 5521 Physical address: 14993 Value: 0
Virtual address: 11225 Physical address: 15321 Value: 0
Virtual address: 41244 Physical address: 15388 Value: 0
Virtual address: 65086 Physical address: 9790 Value: 63
Virtual address: 42927 Physical address: 15791 Value: -21
Virtual address: 22675 Physical address: 16019 Value: 36
Virtual address: 35759 Physical address: 16303 Value: -21
Virtual address: 61845 Physical address: 4501 Value: 0
Virtual address: 47233 Physical address: 12673 Value: 0
Virtual address: 57529 Physical address: 16569 Value: 0
Virtual address: 18576 Physical address: 16784 Value: 0
Virtual address: 15747 Physical address: 17027 Value: 96
Virtual address: 63136 Physical address: 17312 Value: 0
Virtual address: 18190 Physical address: 5646 Value: 17
Virtual address: 58870 Physical address: 17654 Value: 57
Virtual address: 1199 Physical address: 17839 Value: 43
Virtual address: 46394 Physical address: 17978 Value: 45
Virtual address: 45900 Physical address: 18252 Value: 0
Virtual address: 19488 Physical address: 18464 Value: 0
Virtual address: 58916 Physical address: 18724 Value: 0
Virtual address: 39608 Physical address: 19128 Value: 0
Virtual address: 45877 Physical address: 18229 Value: 0
Virtual address: 39736 Physical address: 19256 Value: 0
Virtual address: 42245 Physical address: 8453 Value: 0
Virtual address: 31587 Physical address: 19555 Value: -40
Virtual address: 31774 Physical address: 19742 Value: 31
Virtual address: 39477 Physical address: 18997 Value: 0
Virtual address: 42003 Physical address: 1811 Value: 4
Virtual address: 30990 Physical address: 19982 Value: 30
Virtual address: 13165 Physical address: 13421 Value: 0
Virtual address: 2097 Physical address: 20273 Value: 0
Virtual address: 16108 Physical address: 20716 Value: 0
Virtual address: 39133 Physical address: 20957 Value: 0
Virtual address: 23378 Physical address: 11346 Value: 22
Virtual address: 8601 Physical address: 21145 Value: 0
Virtual address: 64190 Physical address: 8126 Value: 62
Virtual address: 19889 Physical address: 21425 Value: 0
Virtual address: 9079 Physical address: 11639 Value: -35
Virtual address: 62168 Physical address: 21720 Value: 0
Virtual address: 55264 Physical address: 21984 Value: 0
Virtual address: 28126 Physical address: 22238 Value: 27
Virtual address: 8443 Physical address: 22523 Value: 62
Virtual address: 16804 Physical address: 10660 Value: 0
Virtual address: 17393 Physical address: 22769 Value: 0
Virtual address: 10065 Physical address: 22865 Value: 0
Virtual address: 38667 Physical address: 23051 Value: -62
Virtual address: 23491 Physical address: 11459 Value: -16
Virtual address: 27283 Physical address: 14739 Value: -92
Virtual address: 47488 Physical address: 23424 Value: 0
Virtual address: 38126 Physical address: 23790 Value: 37
Virtual address: 62310 Physical address: 2662 Value: 60
Virtual address: 29472 Physical address: 23840 Value: 0
Virtual address: 27909 Physical address: 22021 Value: 0
Virtual address: 426 Physical address: 1706 Value: 0
Virtual address: 8667 Physical address: 21211 Value: 118
Virtual address: 9419 Physical address: 14539 Value: 50
Virtual address: 37953 Physical address: 23617 Value: 0
Virtual address: 13671 Physical address: 24167 Value: 89
Virtual address: 53378 Physical address: 8834 Value: 52
Virtual address: 22850 Physical address: 24386 Value: 22
Virtual address: 33347 Physical address: 9283 Value: -112
Virtual address: 37980 Physical address: 23644 Value: 0
Virtual address: 43202 Physical address: 24770 Value: 42
Virtual address: 63830 Physical address: 4950 Value: 62
Virtual address: 32623 Physical address: 3951 Value: -37
Virtual address: 34823 Physical address: 24839 Value: 1
Virtual address: 16451 Physical address: 25155 Value: 16
Virtual address: 29028 Physical address: 25444 Value: 0
Virtual address: 56987 Physical address: 25755 Value: -90
Virtual address: 26974 Physical address: 25950 Value: 26
Virtual address: 7037 Physical address: 5245 Value: 0
Virtual address: 48481 Physical address: 26209 Value: 0
Virtual address: 37893 Physical address: 23557 Value: 0
Virtual address: 26499 Physical address: 26499 Value: -32
Virtual address: 25042 Physical address: 26834 Value: 24
Virtual address: 57428 Physical address: 16468 Value: 0
Virtual address: 39335 Physical address: 27047 Value: 105
Virtual address: 3949 Physical address: 27245 Value: 0
Virtual address: 23936 Physical address: 27520 Value: 0
Virtual address: 9067 Physical address: 11627 Value: -38
Virtual address: 1489 Physical address: 27857 Value: 0
Virtual address: 64865 Physical address: 28001 Value: 0
Virtual address: 7304 Physical address: 28296 Value: 0
Virtual address: 12092 Physical address: 28476 Value: 0
Virtual address: 44589 Physical address: 28717 Value: 0
Virtual address: 34507 Physical address: 6347 Value: -78
Virtual address: 58377 Physical address: 28937 Value: 0
Virtual address: 34508 Physical address: 6348 Value: 0
Virtual address: 35890 Physical address: 29234 Value: 35
Virtual address: 8597 Physical address: 21141 Value: 0
Virtual address: 45455 Physical address: 29583 Value: 99
Virtual address: 41341 Physical address: 15485 Value: 0
Virtual address: 25192 Physical address: 29800 Value: 0
Virtual address: 43599 Physical address: 30031 Value: -109
Virtual address: 50196 Physical address: 30228 Value: 0
Virtual address: 43755 Physical address: 30187 Value: -70
Virtual address: 54257 Physical address: 30705 Value: 0
Virtual address: 50039 Physical address: 14199 Value: -35
Virtual address: 42355 Physical address: 8563 Value: 92
Virtual address: 11932 Physical address: 30876 Value: 0
Virtual address: 48930 Physical address: 9506 Value: 47
Virtual address: 20987 Physical address: 31227 Value: 126
Virtual address: 49522 Physical address: 31346 Value: 48
Virtual address: 36290 Physical address: 31682 Value: 35
Virtual address: 61858 Physical address: 4514 Value: 60
Virtual address: 48566 Physical address: 26294 Value: 47
Virtual address: 33583 Physical address: 31791 Value: -53
Virtual address: 64067 Physical address: 8003 Value: -112
Virtual address: 17709 Physical address: 32045 Value: 0
Virtual address: 46062 Physical address: 18414 Value: 44
Virtual address: 35677 Physical address: 16221 Value: 0
Virtual address: 25731 Physical address: 32387 Value: 32
Virtual address: 22711 Physical address: 16055 Value: 45
Virtual address: 2240 Physical address: 20416 Value: 0
Virtual address: 39570 Physical address: 19090 Value: 38
Virtual address: 14041 Physical address: 32729 Value: 0
Virtual address: 6065 Physical address: 32945 Value: 0
Virtual address: 8606 Physical address: 21150 Value: 8
Virtual address: 13750 Physical address: 24246 Value: 13
Virtual address: 23976 Physical address: 27560 Value: 0
Virtual address: 11624 Physical address: 5992 Value: 0
Virtual address: 5816 Physical address: 33208 Value: 0
Virtual address: 57488 Physical address: 16528 Value: 0
Virtual address: 14954 Physical address: 874 Value: 14
Virtual address: 27734 Physical address: 33366 Value: 27
Virtual address: 11913 Physical address: 30857 Value: 0
Virtual address: 65155 Physical address: 9859 Value: -96
Virtual address: 57326 Physical address: 4334 Value: 55
Virtual address: 20025 Physical address: 33593 Value: 0
Virtual address: 39374 Physical address: 27086 Value: 38
Virtual address: 23107 Physical address: 33859 Value: -112
Virtual address: 23986 Physical address: 27570 Value: 23
Virtual address: 51334 Physical address: 34182 Value: 50
Virtual address: 15502 Physical address: 34446 Value: 15
Virtual address: 46642 Physical address: 34610 Value: 45
Virtual address: 64120 Physical address: 8056 Value: 0
Virtual address: 30396 Physical address: 35004 Value: 0
Virtual address: 58308 Physical address: 6852 Value: 0
Virtual address: 25094 Physical address: 29702 Value: 24
Virtual address: 4870 Physical address: 35078 Value: 4
Virtual address: 45411 Physical address: 29539 Value: 88
Virtual address: 26726 Physical address: 5478 Value: 26
Virtual address: 21485 Physical address: 35565 Value: 0
Virtual address: 34629 Physical address: 35653 Value: 0
Virtual address: 28196 Physical address: 35876 Value: 0
Virtual address: 1793 Physical address: 36097 Value: 0
Virtual address: 35095 Physical address: 36375 Value: 69
Virtual address: 55637 Physical address: 10069 Value: 0
Virtual address: 52347 Physical address: 36731 Value: 30
Virtual address: 5781 Physical address: 33173 Value: 0
Virtual address: 32969 Physical address: 37065 Value: 0
Virtual address: 41800 Physical address: 37192 Value: 0
Virtual address: 56704 Physical address: 37504 Value: 0
Virtual address: 44254 Physical address: 37854 Value: 43
Virtual address: 16366 Physical address: 494 Value: 15
Virtual address: 65067 Physical address: 9771 Value: -118
Virtual address: 21750 Physical address: 38134 Value: 21
Virtual address: 23204 Physical address: 33956 Value: 0
Virtual address: 18912 Physical address: 38368 Value: 0
Virtual address: 15918 Physical address: 20526 Value: 15
Virtual address: 52866 Physical address: 2434 Value: 51
Virtual address: 48542 Physical address: 26270 Value: 47
Virtual address: 22760 Physical address: 16104 Value: 0
Virtual address: 4037 Physical address: 27333 Value: 0
Virtual address: 11551 Physical address: 5919 Value: 71
Virtual address: 57915 Physical address: 38459 Value: -114
Virtual address: 58716 Physical address: 17500 Value: 0
Virtual address: 27550 Physical address: 38814 Value: 26
Virtual address: 41293 Physical address: 15437 Value: 0
Virtual address: 26083 Physical address: 39139 Value: 120
Virtual address: 29422 Physical address: 39406 Value: 28
Virtual address: 53530 Physical address: 39450 Value: 52
Virtual address: 30369 Physical address: 34977 Value: 0
Virtual address: 1483 Physical address: 27851 Value: 114
Virtual address: 12881 Physical address: 39761 Value: 0
Virtual address: 16006 Physical address: 20614 Value: 15
Virtual address: 40529 Physical address: 40017 Value: 0
Virtual address: 62702 Physical address: 40430 Value: 61
Virtual address: 48037 Physical address: 40613 Value: 0
Virtual address: 811 Physical address: 40747 Value: -54
Virtual address: 58686 Physical address: 17470 Value: 57
Virtual address: 32892 Physical address: 36988 Value: 0
Virtual address: 12364 Physical address: 41036 Value: 0
Virtual address: 33511 Physical address: 9447 Value: -71
Virtual address: 14775 Physical address: 41399 Value: 109
Virtual address: 31559 Physical address: 19527 Value: -47
Virtual address: 42707 Physical address: 41683 Value: -76
Virtual address: 59157 Physical address: 41749 Value: 0
Virtual address: 33822 Physical address: 42014 Value: 33
Virtual address: 58182 Physical address: 6726 Value: 56
Virtual address: 59108 Physical address: 18916 Value: 0
Virtual address: 43638 Physical address: 30070 Value: 42
Virtual address: 34458 Physical address: 6298 Value: 33
Virtual address: 24634 Physical address: 42298 Value: 24
Virtual address: 750 Physical address: 42734 Value: 0
Virtual address: 41056 Physical address: 42848 Value: 0
Virtual address: 64365 Physical address: 109 Value: 0
Virtual address: 36948 Physical address: 43092 Value: 0
Virtual address: 31123 Physical address: 20115 Value: 100
Virtual address: 40942 Physical address: 43502 Value: 39
Virtual address: 42406 Physical address: 8614 Value: 41
Virtual address: 47136 Physical address: 12576 Value: 0
Virtual address: 40919 Physical address: 43479 Value: -11
Virtual address: 48948 Physical address: 9524 Value: 0
Virtual address: 13848 Physical address: 32536 Value: 0
Virtual address: 21168 Physical address: 10928 Value: 0
Virtual address: 46771 Physical address: 34739 Value: -84
Virtual address: 35929 Physical address: 29273 Value: 0
Virtual address: 28601 Physical address: 2233 Value: 0
Virtual address: 40919 Physical address: 43479 Value: -11
Virtual address: 5636 Physical address: 33028 Value: 0
Virtual address: 25092 Physical address: 29700 Value: 0
Virtual address: 27010 Physical address: 25986 Value: 26
Virtual address: 51970 Physical address: 43522 Value: 50
Virtual address: 48231 Physical address: 13671 Value: 25
Virtual address: 59032 Physical address: 18840 Value: 0
Virtual address: 21713 Physical address: 38097 Value: 0
Virtual address: 2516 Physical address: 43988 Value: 0
Virtual address: 22331 Physical address: 44091 Value: -50
Virtual address: 60593 Physical address: 44465 Value: 0
Virtual address: 26305 Physical address: 44737 Value: 0
Virtual address: 55 Physical address: 44855 Value: 13
Virtual address: 11602 Physical address: 5970 Value: 11
Virtual address: 48486 Physical address: 26214 Value: 47
Virtual address: 24084 Physical address: 45076 Value: 0
Virtual address: 25797 Physical address: 32453 Value: 0
Virtual address: 45214 Physical address: 7326 Value: 44
Virtual address: 40843 Physical address: 43403 Value: -30
Virtual address: 62187 Physical address: 21739 Value: -70
Virtual address: 53550 Physical address: 39470 Value: 52
Virtual address: 7096 Physical address: 5304 Value: 0
Virtual address: 9187 Physical address: 11747 Value: -8
Virtual address: 65348 Physical address: 12356 Value: 0
Virtual address: 26539 Physical address: 26539 Value: -22
Virtual address: 35123 Physical address: 36403 Value: 76
Virtual address: 62644 Physical address: 40372 Value: 0
Virtual address: 39192 Physical address: 26904 Value: 0
Virtual address: 19501 Physical address: 18477 Value: 0
Virtual address: 18364 Physical address: 5820 Value: 0
Virtual address: 5316 Physical address: 45508 Value: 0
Virtual address: 65070 Physical address: 9774 Value: 63
Virtual address: 59101 Physical address: 18909 Value: 0
Virtual address: 52246 Physical address: 36630 Value: 51
Virtual address: 37579 Physical address: 45771 Value: -78
Virtual address: 53593 Physical address: 39513 Value: 0
Virtual address: 63705 Physical address: 46041 Value: 0
Virtual address: 54469 Physical address: 46277 Value: 0
Virtual address: 51087 Physical address: 46479 Value: -29
Virtual address: 26533 Physical address: 26533 Value: 0
Virtual address: 9621 Physical address: 46741 Value: 0
Virtual address: 63383 Physical address: 46999 Value: -27
Virtual address: 30841 Physical address: 47225 Value: 0
Virtual address: 33594 Physical address: 31802 Value: 32
Virtual address: 16040 Physical address: 20648 Value: 0
Virtual address: 5917 Physical address: 32797 Value: 0
Virtual address: 21105 Physical address: 10865 Value: 0
Virtual address: 25668 Physical address: 32324 Value: 0
Virtual address: 62026 Physical address: 21578 Value: 60
Virtual address: 32744 Physical address: 4072 Value: 0
Virtual address: 35976 Physical address: 29320 Value: 0
Virtual address: 29197 Physical address: 39181 Value: 0
Virtual address: 50897 Physical address: 47569 Value: 0
Virtual address: 36940 Physical address: 43084 Value: 0
Virtual address: 56216 Physical address: 47768 Value: 0
Virtual address: 12705 Physical address: 48033 Value: 0
Virtual address: 50078 Physical address: 14238 Value: 48
Virtual address: 33022 Physical address: 37118 Value: 32
Virtual address: 60855 Physical address: 48311 Value: 109
Virtual address: 27793 Physical address: 33425 Value: 0
Virtual address: 15089 Physical address: 1009 Value: 0
Virtual address: 48155 Physical address: 13595 Value: 6
Virtual address: 51942 Physical address: 48614 Value: 50
Virtual address: 15287 Physical address: 13239 Value: -19
Virtual address: 51853 Physical address: 48525 Value: 0
Virtual address: 45309 Physical address: 7421 Value: 0
Virtual address: 22290 Physical address: 44050 Value: 21
Virtual address: 36212 Physical address: 31604 Value: 0
Virtual address: 3892 Physical address: 27188 Value: 0
Virtual address: 5559 Physical address: 15031 Value: 109
Virtual address: 61883 Physical address: 4539 Value: 110
Virtual address: 31971 Physical address: 19939 Value: 56
Virtual address: 53744 Physical address: 39664 Value: 0
Virtual address: 59333 Physical address: 41925 Value: 0
Virtual address: 62132 Physical address: 21684 Value: 0
Virtual address: 23466 Physical address: 11434 Value: 22
Virtual address: 54076 Physical address: 30524 Value: 0
Virtual address: 54211 Physical address: 30659 Value: -16
Virtual address: 49400 Physical address: 48888 Value: 0
Virtual address: 57750 Physical address: 12950 Value: 56
Virtual address: 19228 Physical address: 48924 Value: 0
Virtual address: 55031 Physical address: 12023 Value: -67
Virtual address: 11480 Physical address: 49368 Value: 0
Virtual address: 26439 Physical address: 26439 Value: -47
Virtual address: 61997 Physical address: 21549 Value: 0
Virtual address: 4217 Physical address: 6521 Value: 0
Virtual address: 23963 Physical address: 27547 Value: 102
Virtual address: 61987 Physical address: 21539 Value: -120
Virtual address: 10908 Physical address: 49564 Value: 0
Virtual address: 22557 Physical address: 15901 Value: 0
Virtual address: 40688 Physical address: 40176 Value: 0
Virtual address: 19264 Physical address: 48960 Value: 0
Virtual address: 45387 Physical address: 29515 Value: 82
Virtual address: 55275 Physical address: 21995 Value: -6
Virtual address: 14154 Physical address: 49738 Value: 13
Virtual address: 13004 Physical address: 39884 Value: 0
Virtual address: 23018 Physical address: 24554 Value: 22
Virtual address: 54244 Physical address: 30692 Value: 0
Virtual address: 43524 Physical address: 29956 Value: 0
Virtual address: 2447 Physical address: 43919 Value: 99
Virtual address: 48750 Physical address: 50030 Value: 47
Virtual address: 65505 Physical address: 12513 Value: 0
Virtual address: 33617 Physical address: 31825 Value: 0
Virtual address: 2245 Physical address: 20421 Value: 0
Virtual address: 18631 Physical address: 16839 Value: 49
Virtual address: 63130 Physical address: 17306 Value: 61
Virtual address: 22809 Physical address: 24345 Value: 0
Virtual address: 53704 Physical address: 39624 Value: 0
Virtual address: 23564 Physical address: 3340 Value: 0
Virtual address: 42225 Physical address: 2033 Value: 0
Virtual address: 20456 Physical address: 50408 Value: 0
Virtual address: 853 Physical address: 40789 Value: 0
Virtual address: 33393 Physical address: 9329 Value: 0
Virtual address: 16432 Physical address: 25136 Value: 0
Virtual address: 34303 Physical address: 50687 Value: 127
Virtual address: 39187 Physical address: 26899 Value: 68
Virtual address: 60915 Physical address: 48371 Value: 124
Virtual address: 10259 Physical address: 50707 Value: 4
Virtual address: 3988 Physical address: 27284 Value: 0
Virtual address: 57717 Physical address: 12917 Value: 0
Virtual address: 60324 Physical address: 51108 Value: 0
Virtual address: 21946 Physical address: 14010 Value: 21
Virtual address: 58495 Physical address: 29055 Value: 31
Virtual address: 10184 Physical address: 22984 Value: 0
Virtual address: 54538 Physical address: 51210 Value: 53
Virtual address: 53807 Physical address: 51503 Value: -117
Virtual address: 3657 Physical address: 51785 Value: 0
Virtual address: 43799 Physical address: 51991 Value: -59
Virtual address: 11693 Physical address: 6061 Value: 0
Virtual address: 36648 Physical address: 52264 Value: 0
Virtual address: 3458 Physical address: 52610 Value: 3
Virtual address: 49363 Physical address: 48851 Value: 52
Virtual address: 63808 Physical address: 4928 Value: 0
Virtual address: 23335 Physical address: 11303 Value: -55
Virtual address: 34196 Physical address: 50580 Value: 0
Virtual address: 61253 Physical address: 2885 Value: 0
Virtual address: 20814 Physical address: 31054 Value: 20
Virtual address: 13145 Physical address: 13401 Value: 0
Virtual address: 56779 Physical address: 37579 Value: 114
Virtual address: 26966 Physical address: 25942 Value: 26
Virtual address: 32294 Physical address: 52774 Value: 31
Virtual address: 8981 Physical address: 11541 Value: 0
Virtual address: 9973 Physical address: 53237 Value: 0
Virtual address: 34187 Physical address: 50571 Value: 98
Virtual address: 39141 Physical address: 20965 Value: 0
Virtual address: 45427 Physical address: 29555 Value: 92
Virtual address: 2383 Physical address: 43855 Value: 83
Virtual address: 26919 Physical address: 25895 Value: 73
Virtual address: 21340 Physical address: 35420 Value: 0
Virtual address: 1582 Physical address: 1070 Value: 1
Virtual address: 61375 Physical address: 3007 Value: -17
Virtual address: 29458 Physical address: 23826 Value: 28
Virtual address: 45992 Physical address: 18344 Value: 0
Virtual address: 44730 Physical address: 28858 Value: 43
Virtual address: 28542 Physical address: 2174 Value: 27
Virtual address: 11262 Physical address: 15358 Value: 10
Virtual address: 39571 Physical address: 19091 Value: -92
Virtual address: 47387 Physical address: 23323 Value: 70
Virtual address: 57781 Physical address: 12981 Value: 0
Virtual address: 31859 Physical address: 19827 Value: 28
Virtual address: 12719 Physical address: 48047 Value: 107
Virtual address: 64018 Physical address: 7954 Value: 62
Virtual address: 60358 Physical address: 51142 Value: 58
Virtual address: 14307 Physical address: 49891 Value: -8
Virtual address: 49075 Physical address: 9651 Value: -20
Virtual address: 27585 Physical address: 38849 Value: 0
Virtual address: 33142 Physical address: 53366 Value: 32
Virtual address: 21727 Physical address: 38111 Value: 55
Virtual address: 24199 Physical address: 45191 Value: -95
Virtual address: 52045 Physical address: 43597 Value: 0
Virtual address: 54270 Physical address: 30718 Value: 52
Virtual address: 50900 Physical address: 47572 Value: 0
Virtual address: 40311 Physical address: 53623 Value: 93
Virtual address: 48658 Physical address: 49938 Value: 47
Virtual address: 14051 Physical address: 32739 Value: -72
Virtual address: 37756 Physical address: 53884 Value: 0
Virtual address: 1496 Physical address: 27864 Value: 0
Virtual address: 13012 Physical address: 39892 Value: 0
Virtual address: 26447 Physical address: 26447 Value: -45
Virtual address: 4438 Physical address: 54102 Value: 4
Virtual address: 27645 Physical address: 38909 Value: 0
Virtual address: 63763 Physical address: 4883 Value: 68
Virtual address: 46549 Physical address: 18133 Value: 0
Virtual address: 45538 Physical address: 29666 Value: 44
Virtual address: 24612 Physical address: 42276 Value: 0
Virtual address: 26601 Physical address: 26601 Value: 0
Virtual address: 60983 Physical address: 54327 Value: -115
Virtual address: 19961 Physical address: 21497 Value: 0
Virtual address: 27175 Physical address: 14631 Value: -119
Virtual address: 22498 Physical address: 44258 Value: 21
Virtual address: 40372 Physical address: 53684 Value: 0
Virtual address: 6182 Physical address: 10278 Value: 6
Virtual address: 63751 Physical address: 4871 Value: 65
Virtual address: 60182 Physical address: 50966 Value: 58
Virtual address: 53917 Physical address: 51613 Value: 0
Virtual address: 55752 Physical address: 10184 Value: 0
Virtual address: 48730 Physical address: 50010 Value: 47
Virtual address: 10072 Physical address: 22872 Value: 0
Virtual address: 50844 Physical address: 47516 Value: 0
Virtual address: 28006 Physical address: 22118 Value: 27
Virtual address: 11551 Physical address: 5919 Value: 71
Virtual address: 36424 Physical address: 54600 Value: 0
Virtual address: 36813 Physical address: 52429 Value: 0
Virtual address: 58036 Physical address: 38580 Value: 0
Virtual address: 62340 Physical address: 2692 Value: 0
Virtual address: 57562 Physical address: 16602 Value: 56
Virtual address: 64750 Physical address: 55022 Value: 63
Virtual address: 29908 Physical address: 55252 Value: 0
Virtual address: 41388 Physical address: 15532 Value: 0
Virtual address: 54333 Physical address: 46141 Value: 0
Virtual address: 61382 Physical address: 3014 Value: 59
Virtual address: 39496 Physical address: 19016 Value: 0
Virtual address: 14659 Physical address: 41283 Value: 80
Virtual address: 23337 Physical address: 11305 Value: 0
Virtual address: 47765 Physical address: 55445 Value: 0
Virtual address: 42681 Physical address: 41657 Value: 0
Virtual address: 40337 Physical address: 53649 Value: 0
Virtual address: 38501 Physical address: 55653 Value: 0
Virtual address: 50610 Physical address: 55986 Value: 49
Virtual address: 39104 Physical address: 20928 Value: 0
Virtual address: 48015 Physical address: 40591 Value: -29
Virtual address: 16336 Physical address: 464 Value: 0
Virtual address: 10675 Physical address: 56243 Value: 108
Number of Translated Addresses = 500
Page Faults = 220
Page Fault Rate = 0.440
TLB Hits = 32
TLB Hit Rate = 0.064
